src/ButtonCatalog.js

```diff
@@ -91,4 +143,50 @@ class ButtonDemos extends Component {
   }
 }
 
+const ButtonConfig = {
+  afterUpdate: () => {
+    // Function that will be executed after the update. Make any adjustments
+    // such as disabling a field.
+  },
+  options: [
+    {
+      type: 'label',
+      name: 'Options',
+    },
+    {
+      type: 'radiogroup',
+      name: 'Type',
+      urlParam: 'type',
+      value: 'text', // default select first option
```

Comment:
can you set this a default in the HeroComponent? So you don't have to set it here

Reply:
Yeah, I could set the default to be the first option if this is null. Is that what you mean?